fk/lisjp/src/main/java/se/inera/intyg/common/lisjp/v1/rest/LisjpModuleApiV1.java

```diff
@@ -303,4 +310,20 @@ public String getUpdatedJsonWithTestData(String model, FillType fillType, TypeAh
         TestabilityToolkit.fillCertificateWithTestData(certificate, fillType, new LispTestabilityCertificateTestdataProvider());
         return getJsonFromCertificate(certificate, model);
     }
+
+    @Override
+    public CertificateText getPreambleForCitizens() {
+        return CertificateText
+            .builder()
+            .text(PREAMBLE_FOR_CITIZENS)
+            .type(CertificateTextType.PREAMBLE_TEXT)
+            .links(List.of(
+                CertificateLink.builder()
+                    .url("http://www.forsakringskassan.se/sjuk")
```

Comment:
Both url & name could be turned into constants

Reply:
In these large, legacy files I think adding the constants makes it harder to read the file and harder to understand where the values are supposed to be used. Especially in these cases where the values aren't used anywhere else in the file and the Strings are short I think they can belong in the creation of the object. But I can update it.